Source/Core/Core/NetPlayClient.h

```diff
@@ -68,11 +66,25 @@ class NetPlayClient
 	bool WiimoteUpdate(int _number, u8* data, const u8 size);
 	bool GetNetPads(const u8 pad_nb, GCPadStatus* pad_status);
 
+	virtual void OnTraversalStateChanged();
+	virtual void OnConnectReady(ENetAddress addr);
+	virtual void OnConnectFailed(u8 reason);
```

Comment:
`virtual` can be removed. Add `override` to the end of the function. Like so:

`void OnTraversalStateChanged() override;`